integration-tests/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   preset: 'jest-puppeteer',
   testEnvironment: '<rootDir>/jest-circus-environment.js',
   setupFilesAfterEnv: ['<rootDir>/jest.test-setup.js'],
-  testMatch: ['<rootDir>/tests/*.js'],
+  testMatch: ['<rootDir>/tests/*.js', '<rootDir>/tests/*.ts'],
```

Comment:
Alternatively, this can be written so that the path only has to be specified once.

```suggestion
  testMatch: ['<rootDir>/tests/*.{js,ts}'],
```

```suggestion
  testMatch: ['<rootDir>/tests/*.[jt]s'],
```

https://jestjs.io/docs/configuration#testmatch-arraystring

Reply:
I like js|ts, I will change that